text/dependencies/rfcs/0000-dependency-management-phase-one.md

```diff
@@ -0,0 +1,437 @@
+| dotnet-sdk        | https://download.visualstudio.microsoft.com/download/pr   | processed, buildpacks-ci                 |
+| go                | https://dl.google.com/go/                                 | yes, binary-builder                      |
+| httpd             | http://archive.apache.org/dist/httpd                      | yes, binary-builder                      |
+| icu               | https://github.com/unicode-org/icu/releases/download/     | yes, buildpacks-ci                       |
+| nginx             | http://nginx.org/download/                                | yes, both locations?                     |
+| node              | https://nodejs.org/dist/                                  | yes, binary-builder                      |
+| php               | https://www.php.net/distributions/                        | yes, binary-builder                      |
+| pip               | https://files.pythonhosted.org/packages                   | yes, with other pip deps buildpacks-ci   |
+| pipenv            | https://files.pythonhosted.org/packages                   | yes, with other pip deps buildpacks-ci   |
+| python            | https://www.python.org/ftp/python                         | yes, buildpacks-ci                       |
+| poetry            | https://files.pythonhosted.org/packages                   | processed, buildpacks-ci                 |
+| ruby              | https://cache.ruby-lang.org/pub/ruby                      | yes, binary-builder                      |
+| rust              | https://static.rust-lang.org/dist/                        | yes, buildpacks-ci                       |
+| tini              | https://github.com/krallin/tini/tarball                   | yes, buildpacks-ci                       |
+| yarn              | https://github.com/yarnpkg/yarn/releases/download/v1.15.2 | processed, buildpacks-ci                 |
+</details>
+
+### Transfer Compilation Code
+
+For any dependencies that must still be compiled or processed in some way, the
+code from the Cloud Foundry repositories should be rewritten,
+and moved over to the buildpack under a directory called
+`dependency/compilation`. The code should be easily runnable from a local
+machine to compile the code, and should be documented with a README about how
+to use it. The dependency will likely need to be compiled against different
+stacks/architectures, so this should also be enumerated in the RFC and taken
+account of in the new compilation code.
+
+The code will eventually be used in a Github Actions workflow, so the location
+needs to be standardized across buildpacks. The workflows and Github Actions to
+use this code will be enumerated in a separate RFC.
+
+### Bucket Setup
+
+For dependency metadata and the dependencies that must be compiled, there
+should be a dependency-specific bucket set up in Paketo Dependencies project
```

Comment:
Can we at least make this less specific? We won't be making a new Dependencies project, and it might ultimately no end up on GCP at all. I feel like we don't need to be this specific here.

Reply:
Sure.

Reply:
https://github.com/paketo-buildpacks/rfcs/pull/205/commits/9136ab0ec53c631579b8fc66953e978dcda13820